CMakeLists.txt

```diff
@@ -67,7 +83,7 @@ if(NOT SH_EXE)
 	message(FATAL_ERROR "sh interpreter was not found in your path,please install one.On Windows you can get it from here https://gitforwindows.org/")
 endif()
 
-if(WIN32)
+if(WIN32 AND NOT MSVC)#not required for visual studio builds
```

Comment:
I would rather include the `git.rc` explicitly, even with Visual Studio, so that the version information is recorded correctly in the `.exe` files. But maybe on Visual Studio, we do not need to compile the file with `WINDRES_EXE`, is that the problem?

Reply:
Ah, I read further down that you add `git-rc` as a dependency. Is that how this is handled? If so, a comment over here would do wonders to stave off puzzlement like mine ;-)

Reply:
Using git.rc generates link warnings when building with MSVC as it generates it's own resource information. Will look into fixing this.

Reply:
It may have to be specified differently from `*.c` files.

Reply:
What was the resolution? Could it be explained in the commit message?